front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -113,6 +114,31 @@ const EventInformation = ({
               </Text>
             </Box>
           )}
+          {onlineLink && (
+            <Box display="flex" mb="12px">
+              <Icon
+                my="auto"
+                fill={colors.coolGrey300}
+                name="link"
+                ariaHidden
+                mr="8px"
+              />
+              <Text
+                m="0px"
+                color={'coolGrey700'}
+                fontSize="s"
+                role="button"
+                onClick={(e) => {
+                  e.preventDefault();
```

Comment:
In the case here, this Text component is within the "EventCard" component, which itself is a clickable card (like the idea cards). But when we click this Text component, we don't want to also click the card itself. So we need to prevent the default + propogation so that doesn't happen :)

Reply:
TIL